tests/test_psfDetermination.py

```diff
@@ -258,7 +258,7 @@ def subtractStars(self, exposure, catalog, chi_lim=-1):
             if bbox.contains(afwGeom.PointI(int(xc), int(yc))):
                 try:
                     measAlg.subtractPsf(psf, subtracted, xc, yc)
-                except:
+                except Exception:
```

Comment:
Please check if there's an obvious exception(s) raised by subtractPSf here (as above).

Reply:
I agree that this is a very nasty spot to ignore errors. I tried leaving out the try/except to see what got raised and nothing did, so I removed it.